security-analysis/src/main/java/com/powsybl/security/SecurityAnalysisResult.java

```diff
@@ -14,16 +16,31 @@
  */
 public class SecurityAnalysisResult {
 
+    private final Network network;
```

Comment:
Not a good idea to put network here as you wont be able after a json loop to recreate it.

Reply:
The network is skipped by the exporter. I’ll try to find a better way to handle the network